docs/pages/versions/unversioned/sdk/filesystem-next.mdx

```diff
@@ -0,0 +1,23 @@
+---
+title: FileSystem (next)
+description: A next version of a library that provides access to the local file system on the device.
```

Comment:
```suggestion
description: A library that provides access to the local file system on the device.
```

Instead of using "next" in the reference's description, let's use the `PrereleaseNotice` component on the page: https://github.com/expo/expo/blob/72b3295ce4d2030f6e5fe1d3391236d9e6c0bdd6/docs/components/PrereleaseNotice.tsx#L7. This collapsible helps explaining what are "next" version of a library are meant for.